src/components/skip-link/default.md.njk

```diff
@@ -0,0 +1,11 @@
+---
+layout: layout-example.njk
+---
+<p class="govuk-body">To view the Skip link component click inside this example and press tab</p>
+
```

Comment:
I would remove this empty line because introduces an extra line between `<p>` and `<a>` on the HTML example
![screen shot 2018-01-04 at 16 58 39](https://user-images.githubusercontent.com/788096/34574557-ab402f18-f16f-11e7-8173-d58f2d30bff0.png)